packages/zookeeper/spec.yml

```diff
@@ -0,0 +1,31 @@
+---
+name: zookeeper
+version: 3.4.6
```

Comment:
On Thu, May 26, 2016 at 06:53:19AM -0700, Steven Borrelli wrote:

> > +---
> > +name: zookeeper
> > +version: 3.4.6
> 
> Can we update this to 3.4.8? 
> 

Yes, I just started from same version as mesosphere's, because
I adapting their makefile/scripts. Will update version/hash
on last step.